### ISSUE TITLE
As a user with a disability, I should be able to use the form to upload a new flyer using a screen reader.

### ISSUE THREAD

**Issue by dmac1108** [dmac1108/a-flyer-on-the-wall-client#5]
Estimate to Complete: 30 minutes
Add appropriate aria elements to form
